core/dataset.cc

```diff
@@ -336,6 +336,21 @@ getColumnValues(const ColumnName & column,
     return std::move(result);
 }
 
+std::vector<CellValue>
+ColumnIndex::
+getColumnDense(const ColumnName & column) const
+{
+    throw HttpReturnException(500, "Dataset type doesn't implement getColumnDense()");
```

Comment:
Is there a way to output the dataset type here?  That would make debugging easier when working with several dataset type as the same time.  Also, I believe this should be a 400 because the user is able to perform an action to fix the issue unless another part of the code already throw a 400.